test/sanity/validate-modules/main.py

```diff
@@ -1020,6 +1032,50 @@ def _validate_argument_spec(self):
                          'should not be marked as required' % arg)
                 )
 
+        if docs:
+            try:
+                add_fragments(docs, self.object_path)
+            except Exception:
+                # Cannot merge fragments
```

Comment:
The code does seem to work for `extends_documentation_fragment`.

What does fragments mean in this context, can the comment be fleshed out a bit?

Reply:
This should work, unless it was as a result of the "temporary" PR being merged, which changed the signature of `add_fragments` slightly, that could have caused this to not work.

I've compared the docs of a module before and after this call, and confirmed that the fragments are properly merged